pkg/informer/informer_test.go

```diff
@@ -126,6 +126,50 @@ func Test_listAndResync_syncWithPrevState(t *testing.T) {
 	if *deleteFuncCount != 1 {
 		t.Errorf("Expected informer.DeleteFunc to be called 1 time, but got %d.", *deleteFuncCount)
 	}
+
+}
+
+func Test_StoppedInformer_ValidateDeleteFunc(t *testing.T) {
+	var uid string
+
+	//create informer for mock resource
+	informer, _, _, _ := initInformer()
+
+	//mock informer state
+	informer.resourceIndex["id-999"] = "12345"
+	informer.resourceIndex["id-100"] = "54321"
+
+	//keep track of calls made to mock informer.DeleteFunc below
+	deleteFuncCalls := make([]string, 0)
+
+	// mock DeleteFunc and get uids
+	informer.DeleteFunc = func(obj interface{}) {
+		for key := range informer.resourceIndex {
+			obj := newUnstructured(informer.gvr.Resource, key)
+			uid = string(obj.GetUID())
```

Comment:
Remove `var uid string` and use `:=` instead.
```suggestion
			uid := string(obj.GetUID())
```